db/evidences.go

```diff
@@ -0,0 +1,32 @@
+package db
+
+import (
+	"github.com/flanksource/incident-commander/api"
+	"github.com/google/uuid"
+
+	"github.com/flanksource/commons/logger"
+)
+
+func GetEvidenceScripts() []api.Evidence {
+	var evidences []api.Evidence
+	openIncidentsSubQuery := Gorm.Table("hypotheses").Select("id").Where("incident_id IN (?)",
+		Gorm.Table("incidents").Select("id").Where("closed IS NULL AND resolved IS NULL"))
```

Comment:
```suggestion
		Gorm.Table("incidents").Select("id").Where("closed IS NULL"))
```

A resolved incident could be re-opened if the script starts failing